manifests/06_service-catalog-controller-manager_cr.yaml

```diff
@@ -3,11 +3,6 @@ kind: ServiceCatalogControllerManager
 metadata:
   name: cluster
 spec:
-  managementState: Managed
-  logLevel: Debug
+  logLevel: "Normal"
+  managementState: Removed
```

Comment:
same issue as the api operator PR:  you can't put this in manifests because the CVO will start managing it for you and prevent people from changing it.

Reply:
here's how the other library-go based operators do this:
https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/bindata/v3.11.0/openshift-controller-manager/operator-config.yaml

https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/83f2aadd472d694938541f1beca05d5ae6298208/pkg/operator/starter.go#L48

Reply:
Awesome, you saved me some hunting.  Thanks much Ben!